libs/ngxtension/active-element/src/lib/active-element.ts

```diff
@@ -0,0 +1,20 @@
+import { DOCUMENT } from '@angular/common';
+import { assertInInjectionContext, inject, type Injector } from '@angular/core';
+import { fromEvent, map, merge, shareReplay } from 'rxjs';
+
+export function injectActiveElement(injector?: Injector) {
+	injector ?? assertInInjectionContext(injectActiveElement);
+	const doc = injector ? injector.get(DOCUMENT) : inject(DOCUMENT);
+
+	return merge(
+		fromEvent(doc, 'focus', { capture: true, passive: true }).pipe(
+			map(() => true)
+		),
+		fromEvent(doc, 'blur', { capture: true, passive: true }).pipe(
```

Comment:
question: would it make sense to allow the consumers to configure the event options?

Reply:
EvenListener has 3 options. `capture`, `passive`, and `once`. In this case, 

`capture` must always be set to true; otherwise, it won't work. 
It is advisable to keep `passive` set to true.
Setting `once` to true can disrupt our intended functionality.

Therefore, I would argue that allowing users to edit these event options is unnecessary.